pynumaflow/tests/function/test_server.py

```diff
@@ -0,0 +1,157 @@
+import json
```

Comment:
The server also has async functions right? We need to have tests for those also

Reply:
as mentioned above https://github.com/numaproj/numaflow-python/pull/4#discussion_r970150907